tests/e2e-agents/e2e_agents/ci_entrypoint.py

```
@@ -0,0 +1,207 @@
+
+        # Limit to 30 files to keep prompt reasonable
+        if len(relevant) > 30:
+            relevant = relevant[:30] + [f"... and {len(relevant) - 30} more"]
+
+        return "\n".join(relevant)
+
+    except Exception:
+        return None
+
+
+async def main():
```

Ruff (0.15.12)

[warning] Too many branches (14 > 12)

(PLR0912)